packages/foreman/foreman-release/foreman-release.spec

```diff
@@ -24,26 +40,44 @@ Foreman repository contains open source and other distributable software for
 distributions in RPM format. This package contains the repository configuration
 for Yum.
 
+%package   client
+Summary:   Foreman client repositories meta-package
+Group:     Applications/System
+
+%description client
+Defines yum repositories for Katello clients.
+
+%files client
+%config %{repo_dir}/foreman-client.repo
+%{_sysconfdir}/pki/rpm-gpg/RPM-GPG-KEY-foreman
```

Comment:
Isn't this a duplication? We also don't sign the client repo so this shouldn't be needed

Reply:
Duplications are allowed. We don't sign the client repo but should we?

Reply:
@ekohl question for oyu here

Reply:
We don't sign plugins because it's an automated process. If we follow that same logic then it'll depend on the release process for client repos. We could also look at automated signing (with a different key than the manually signed packages). I'd suggest starting with an automated process like plugins and unsigned. See where it goes from there.